fdbserver/include/fdbserver/ProxyCommitData.actor.h

```diff
@@ -407,6 +336,18 @@ struct ProxyCommitData {
 		}
 	}
 
+	// RangeLock feature currently does not support version vector
+	// So, if the version vector is enabled, the RangeLock is automatically disabled
+	// RangeLock feature currently rely on processing private mutations in commit proxy
+	// So, if PROXY_USE_RESOLVER_PRIVATE_MUTATIONS is on, the RangeLock is automatically disabled
+	// RangeLock feature currently is not compatible with encryption and tenant
```

Comment:
Add TODO to copy logic to handle range lock from CP to resolver so vv will be compatible with this